bedrock/firefox/templates/firefox/browsers/mobile/focus.html

```diff
@@ -17,6 +17,8 @@
 {% set show_firefox_app_store_banner = switch('firefox-app-store-banner') %}
 {% set show_send_to_device = LANG in settings.SEND_TO_DEVICE_LOCALES %}
 {% set android_url = focus_adjust_url('android', 'mobile-focus-page') %}
+{% set variation_android_adjust_url = 'https://app.adjust.com/167k4ih?campaign=firefox-desktop&adgroup=pb&creative=focus-omc171-uk&redirect=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dorg.mozilla.focus' %}
+{% set variation_ios_adjust_url = 'https://app.adjust.com/167k4ih?campaign=firefox-desktop&adgroup=pb&creative=focus ![qr-code](https://user-images.githubusercontent.com/2030656/162119789-a4541259-76b6-4096-bddf-61867cb46f6e.svg) -omc171-uk&redirect=https%3A%2F%2Fapps.apple.com%2Fus%2Fapp%2Ffirefox-focus-privacy-browser%2Fid1055677337' %}
```

Comment:
This URL seems malformed? Maybe a wayward paste in the issue. 
```suggestion
{% set variation_ios_adjust_url = 'https://app.adjust.com/167k4ih?campaign=firefox-desktop&adgroup=pb&creative=focus-omc171-uk&redirect=https%3A%2F%2Fapps.apple.com%2Fus%2Fapp%2Ffirefox-focus-privacy-browser%2Fid1055677337' %}
```

Reply:
ah good catch, i'l double check with Dan before its ready